Example/ViewController.swift

```diff
@@ -599,7 +599,7 @@ class ViewController: UIViewController {
         return MapboxNavigationService(routeResponse: response,
                                        routeIndex: routeIndex,
                                        routeOptions: options,
-                                       routingProvider: MapboxRoutingProvider(.hybrid),
+                                       routingProvider: nil,
```

Comment:
Is this change indicative of a change that the developer is now expected to make? If so, please describe it in the changelog and also remember to follow up in the mapbox-navigation-ios-examples repository.

Reply:
Added note to the changelog about meaning of `routingProvider` and it's `nil` value